rules/Proj1702.md

```diff
@@ -0,0 +1,28 @@
+# Proj1701: Omit XML declarations
+The XML Declaration provides basic information about the format for the rest of
+the XML document. It takes the form of a Processing Instruction and can have
+the attributes version, encoding and standalone. All of this is redunent for
```

Comment:
What is the "standalone"?

Reply:
It means that the data in the XML does not need external sources (other XML/schema) to be understood. Also totally irrelevant in this context.